include/SetupDialog.h

```diff
@@ -116,10 +116,10 @@ private slots:
 	void setVSTDir(const QString & vstDir);
 	void openLADSPADir();
 	void setLADSPADir(const QString & ladspaDir);
-	void openSF2Dir();
-	void setSF2Dir(const QString & sf2Dir);
-	void openSF2File();
-	void setSF2File(const QString & sf2File);
+	void openSoundFontDir();
+	void setSoundFontDir(const QString & SoundFontDir);
+	void openSoundFontFile();
+	void setSoundFontFile(const QString & soundFontFile);
```

Comment:
(Non-blocking) Per the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
	void setSoundFontDir(const QString& SoundFontDir);
	void openSoundFontFile();
	void setSoundFontFile(const QString& soundFontFile);
```